### ISSUE TITLE
Can't import Keras model

### ISSUE THREAD

**Issue by Cocoa-Puffs** [deeplearning4j/deeplearning4j#8721]
#### Issue Description

I'm working on a project that has machine learning elements in it. One of my colleagues has trained a model and gave me the .h5 file to test. I'm trying to import it using the 

KerasModelImport.importKerasSequentialModelAndWeights(String filePath)

method call. However the import failes with the stack I've provided further down below in a gist.

I've looked into it a bit and it seems that when the method 

KerasBatchNormalization.getBatchNormAxis

tries to cast the innerConfig's axis element into an Integer it fails because axis is an ArrayList with a single element inside it.

#### Version Information

I'm using the Deeplearning4j-core and Deeplearning4j-modelimport packages, both at verison number 1.0.0-beta6. The OS I'm running is Kubuntu 18.04

#### Additional Information

[The StackTrace](https://gist.github.com/Cocoa-Puffs/f8fb7243a9fedb81805dc3a2f5e61886)

I cannot release the pom file of the project.

**Comment by treo** [comment]
What version of keras was used to train that model?

**Comment by Cocoa-Puffs** [comment]
I'm told it was TensorFlow 2.0

**Comment by treo** [comment]
Unfortunately we can't import keras models that are built with tf.keras (neither can multi-backend Keras) at the moment, as they've changed the model format.

**Comment by Cocoa-Puffs** [comment]
Alright, thank you. I will close the issue.
